### DISCUSSION TITLE
Nourrir un familier dans l’abri

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2578] [Suggestions fermées]
Ajouter une option "emplacement :" à la commande /nourrirfamilier pour pouvoir nourrir un animal directement dans l'abri.

Évite de prendre l'animal dans l'abri, le nourrir, le remettre.

**Comment by Cirpacha** [comment]
https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/2513

**Reply by OrionYp** [reply]
Ah oui désolé je ne l'ai pas vue quand j'ai remonté la liste 😅😅😅

**Comment by BastLast** [comment]
Je ferme as duplicate
